azure-mock/Dockerfile

```diff
@@ -0,0 +1,16 @@
+# This file is used to build the docker image for the azure-mock service
+# It is only used for local development and testing purposes
+# This file needs a build context of the root of the repository
+FROM node:20.13-alpine
+
+WORKDIR /app
+
+COPY package*.json ./
+
+RUN npm ci --no-scripts
+
+COPY azure-mock/ ./
+
+ENTRYPOINT ["npx", "tsx", "upload-documents.ts"]
+
+CMD ["data/"]
```

Comment:
What does this command?

Reply:
it provides the script the directory of the data to upload to the blob storage. I made this a CMD rather then adding it to an entry point as it can be overwritten when running the container by hand.